git_machete/git_operations.py

```diff
@@ -733,11 +733,17 @@ def matches(remote_branch: RemoteBranchShortName) -> bool:
         return matching_remote_branches[0] if len(matching_remote_branches) == 1 else None
 
     def get_merged_local_branches(self) -> List[LocalBranchShortName]:
-        return list(map(
-            lambda branch: LocalBranchFullName.of(branch).to_short_name(),
-            utils.get_non_empty_lines(
-                self._popen_git("for-each-ref", "--format=%(refname)", "--merged", "HEAD", "refs/heads"))
-        ))
+        if self.get_git_version() >= (2, 7, 6):  # earliest version of git to support 'for-each-ref --merged'
+            return list(
+                map(lambda branch: LocalBranchFullName.of(branch).to_short_name(),
+                    utils.get_non_empty_lines(
+                        self._popen_git("for-each-ref", "--format=%(refname)", "--merged", "HEAD", "refs/heads"))))
+        else:
```

Comment:
Sanity checking here, are both cases covered in the CI? (i.e. enough if just this method is covered by the tests, then one case will be covered under gits 1.8.0 and 2.7.6 and the other by the later gits)

Reply:
Yes, this is covered by the `test_clean` test